### PR TITLE
chore: build helper tool that can be used when working with all prebuilt providers

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: tools/setup-prebuild-provider-folder.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+set -ex
+
+# Creates a folder with all pre-built providers and lerna packages
+# that can be used to run a command against each (e.g. triggering a workflow or pushing a commit)
+# Requires the gh CLI and jq to be installed
+
+PATH_TO_CREATE=$1
+if [ -z "$PATH_TO_CREATE" ]; then
+    echo "No argument supplied, please call with the path to create the folder"
```
Comment (ansgarm) [inline tools/setup-prebuild-provider-folder.sh:11]: Could be a bit more descriptive (i.e. "Which folder?")

File: tools/setup-prebuild-provider-folder.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+cd packages
+
+# Clone all the repositories, could be parallelized, but it's not worth it
+for repo in $REPO_NAMES; do
+    echo "Adding $repo"
+    
+    gh repo clone "hashicorp/$repo"
+done
+
+# Removing is easier than filtering
```
Comment (ansgarm) [inline tools/setup-prebuild-provider-folder.sh:54]: 👏

File: tools/setup-prebuild-provider-folder.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -ex
+
+# Creates a folder with all pre-built providers and lerna packages
+# that can be used to run a command against each (e.g. triggering a workflow or pushing a commit)
+# Requires the gh CLI and jq to be installed
```
Comment (ansgarm) [inline tools/setup-prebuild-provider-folder.sh:7]:
```suggestion
# Requires the Github CLI (gh) and jq to be installed
```

**Review by ansgarm — APPROVED** [review]
nice!

File: tools/setup-prebuild-provider-folder.sh [hunk 4]
```diff
@@ -48,8 +48,9 @@ cd packages
 for repo in $REPO_NAMES; do
     echo "Adding $repo"
     
-    gh repo clone "hashicorp/$repo"
+    gh repo clone "hashicorp/$repo" &
```
Comment (ansgarm) [inline tools/setup-prebuild-provider-folder.sh:51]: Does the for-loop wait for this to complete? Or does this race with the `rm` commands below?

Comment (DanielMSchmidt) [reply tools/setup-prebuild-provider-folder.sh:52]: yeah, added a wait now that waits for all open jobs

Comment (ansgarm) [reply tools/setup-prebuild-provider-folder.sh:52]: nice! TIL that `wait` helps with this 💡

**Pull request merged**